core/templates/dev/head/pages/error/error.html

```diff
@@ -67,10 +67,8 @@ <h2 class="oppia-error-h2" translate="I18N_ERROR_MESSAGE_500"></h2>
     .oppia-footer {
       position: absolute;
     }
-    @media (max-width: 768px) {
-      .oppia-footer {
-        position: relative;
-      }
+    .oppia-wide-panel-content {
+      margin-bottom: 15em
```

Comment:
Also, setting ` margin-bottom: 15em` is visually appealing but design wise it's bit unsymmetrical. 
As shown below it might result in errors at different resolutions. 

Something like `margin: 70px auto` should be the way to go

![screenshot from 2017-05-23 17-23-37](https://cloud.githubusercontent.com/assets/24438869/26353213/ed75ba96-3fdc-11e7-83a9-d0c89501c025.png)